src/config.js

```diff
@@ -228,11 +360,17 @@ const stripeSupportedCountries = [
     // United States
     code: 'US',
     currency: 'USD',
-    payoutAddressRequired: false,
+    addressConfig: {
+      addressLine: true,
+      city: true,
+      postalCode: true,
+      state: true,
+    },
     accountConfig: {
       routingNumber: true,
       accountNumber: true,
     },
+    personalIdNumberRequired: true,
```

Comment:
Try again with ssn_last_4: '0000'. Test environment might require it to be exactly that.
https://stripe.com/docs/connect/testing#test-personal-id-numbers